example/revisions.trig

```diff
@@ -0,0 +1,78 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix exdata: <https://example.com/data/>.
+@prefix rec: <https://rdf.equinor.com/ontology/record/>.
+@prefix rev: <https://rdf.equinor.com/ontology/revision/>.
+@prefix famdata: <https://rdf.equinor.com/fam/>.
+@prefix contract: <https://rdf.equinor.com/fam/Contract/>.
+@prefix document: <https://rdf.equinor.com/fam/Document/>.
+@prefix facility: <https://rdf.equinor.com/fam/Facility/>.
+@prefix project: <https://rdf.equinor.com/fam/Project/>.
+@prefix fsystem: <https://rdf.equinor.com/fam/FunctionalSystem/>.
+@prefix famvocab: <https://rdf.equinor.com/ontology/fam/v1/>.
+@prefix prov: <http://www.w3.org/ns/prov#>.
+
+exdata:RecordID123 {
+    exdata:RecordID123 a rec:Record;
+                        rdfs:comment "Container record, defines scope for subrecords"^^xsd:string;
+                        prov:generatedAtTime "2022-11-03"^^xsd:date;
+                        rec:isInScope contract:12345678,
+                                    document:B123-EX-W-LA-00001,
+                                    facility:EXMPL,
+                                    project:L.01234A.001.
+}
+
+exdata:RecordID123.0 {
+    exdata:RecordID123.0 a rec:Record;
+                        rdfs:comment "First MEL delivered on Example Facility"^^xsd:string;
+                        prov:generatedAtTime "2022-11-03"^^xsd:date;
+                        rec:describes document:B123-EX-W-LA-00001;
+                        rec:isSubRecordOf exdata:RecordID123.
+    document:B123-EX-W-LA-00001 a <https://rdf.equinor.com/ontology/mel/EquipmentList>;
+                                rdfs:label "B123-EX-W-LA-00001"^^xsd:string;
+                                famvocab:DocumentType "LA"^^xsd:string;
+                                famvocab:Facility facility:EXMPL;
+                                famvocab:FunctionalSystem fsystem:ENG;
+                                famvocab:Location "Northsea"^^xsd:string;
+                                famvocab:Title "MASTER EQUIPMENT LIST - MEL"^^xsd:string.
```

Comment:
The naming is a work in progress, but I think we should at least cover what's in the TIE message today:
                INST_CODE
                PROJECT_CODE
                CONTR_CODE
                DOC_NO
                DOC_TITLE
                **REV_NO**
                DOC_CATEGORY
                DISCIPLINE_CODE
                DOC_TYPE
                **REV_STATUS**
                **REV_DATE**
                **REASON_FOR_ISSUE**
                DOC_GR
                **PURPOSE_CODE**
                CONTRACT_NO
                TRANSFER_CODE

The **bold** codes belongs on the DocumentRevision object. 

PS: I do not suggest using the names above. This is just what they are known as in the TIE_MESSAGE

Reply:
Agree again, but I need to check if that should be in a separate repo

Reply:
Date and Number are covered. I think status and purpose code should be subclasses or object relation. Will cover in new PR